flake.nix

```diff
@@ -0,0 +1,59 @@
+{
+  description = " A fast usermode x86 and x86-64 emulator for Arm64 Linux";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }:
+    let
+      supportedSystems = [ "x86_64-linux" "aarch64-linux" ];
+      forAllSystems = nixpkgs.lib.genAttrs supportedSystems;
+      nixpkgsFor = forAllSystems (system: import nixpkgs { inherit system; });
+
+    in {
+      packages = forAllSystems (system:
+        let
+          pkgs = nixpkgsFor.${system};
+          deps = with pkgs; [
+            git
+            cmake
+            ninja
+            clang
+            binutils
+            lld
+            llvm
+            SDL2
+            libepoxy
+            nasm
+            pkg-config
+            python3
+            gcc-unwrapped
+            python311Packages.setuptools
+          ];
+        in {
+          fex = pkgs.stdenv.mkDerivation rec {
```

Comment:
Can't you use clangStdenv here to avoid the need to set cc and manually install llvm?